calico_node/filesystem/licenses/README.md

```diff
@@ -103,7 +103,7 @@
 |	[docker/distribution](https://github.com/docker/distribution)	|	[Apache License 2.0](http://github.com/docker/distribution/blob/cd27f179f2c10c5d300e6d09025b538c475b0d51/LICENSE)	|
 |	[google/gofuzz](https://github.com/google/gofuzz)	|	[Apache License 2.0](http://github.com/google/gofuzz/blob/bbcb9da2d746f8bdbd6a936686a0a6067ada0ec5/LICENSE)	|
 |	[awslabs/aws-sdk-go](https://github.com/awslabs/aws-sdk-go)	|	[Apache License 2.0](http://github.com/awslabs/aws-sdk-go/blob/29717a72a2fc649e790ce97dc2c4d96e32950844/LICENSE.txt)	|
-|	[Sirupsen/logrus](https://github.com/Sirupsen/logrus)	|	[MIT License](http://github.com/Sirupsen/logrus/blob/092eda23b5d2ccf95fe7dfc70354493eb273487a/LICENSE)	|
+|	[sirupsen/logrus](https://github.com/sirupsen/logrus)	|	[MIT License](http://github.com/sirupsen/logrus/blob/092eda23b5d2ccf95fe7dfc70354493eb273487a/LICENSE)	|
```

Comment:
ah I think we've got a duplicate entry in this table. There should only be one logrus license :) 
(it might be outside of the scope of this PR, but it's a small change you might want to include inthis PR? it's just removing this line)